package org.mobilestoreapp.commons.email;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.stereotype.Service;

@Service
public class ConsoleEmailService implements EmailService {
    private static final Logger log = LoggerFactory.getLogger(ConsoleEmailService.class);
    @Override
    public void send(String to, String subject, String body) {
        log.info("Sending email to={} subject={} body={}", to, subject, body);
    }
}